### PR TITLE
Add optional generation for a package data (setup.py and others)

### LINKED ISSUE #10: Generate setup.py or alternatives
We want to try this generator, but in our case generated client should be published to PyPI in one of CI/CD steps. I see that this project in the proof-of-concept stage, so I also can be helpful ✌️

How can I add setup.py to a generated client?

### PR DESCRIPTION
**Pull request by b0g3r**
Close #10 

@dmontagu, thank you for your awesome code-generator tool, it's perfectly matched with our requirements 🙌

I add one important feature — package metadata generation for ability to publish a package to PyPI, could you check it, please?

P.S> I also found that the example folder is pretty old and doesn't pass the checks. Should I update it?

### CODE REVIEW

File: openapi-python-templates/requirements.mustache [hunk 1]
```diff
@@ -0,0 +1,4 @@
+pydantic >= 1.0
```
Comment (b0g3r) [inline openapi-python-templates/requirements.mustache]: I'm not sure about specific versions, and in `setup.py` I didn't fix them at all. If you know minimal versions for code generated client, please, tell me and I'll change it.

Comment (fj-sanchez) [reply openapi-python-templates/requirements.mustache]: I think I made this change, pydantic introduced some backward incompatible changes so in order to update the rest of the dependencies this was required, have a look at my previous MR.

File: openapi-python-templates/requirements.mustache [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+httpx >= 0.10
+fastapi >= 0.40
```
Comment (fj-sanchez) [inline openapi-python-templates/requirements.mustache]: Initially this was set to 0.42.0, so I wouldn't go lower than that unless you want to do the extra check. Also, I wish if we would get rid of this dependency...

File: openapi-python-templates/travis.mustache [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# ref: https://docs.travis-ci.com/user/languages/python
```
Comment (fj-sanchez) [inline openapi-python-templates/travis.mustache]: Isn't this a very opinionated generation? Why not CircleCI or Gitlab? I would personally leave this out...

Comment (b0g3r) [reply openapi-python-templates/travis.mustache]: It's from standard python client generator. I'll additionally check whether this is necessary

Comment (fj-sanchez) [reply openapi-python-templates/travis.mustache]: Oh I see, well I guess it doesn't hurt to have this, probably better than nothing and might even spark inspiration for other collaborators to provide further implementations.

File: openapi-python-templates/requirements.mustache [hunk 4]
```diff
@@ -0,0 +1,4 @@
+pydantic >= 1.0
+httpx >= 0.10
```
Comment (fj-sanchez) [inline openapi-python-templates/requirements.mustache]: I just created another PR to constrain this dependency as the latest 0.12 version has some backward incompatible changes with the way imports should be done.

**Pull request merged**